vllm/v1/core/kv_cache_manager.py

```diff
@@ -70,6 +70,10 @@ def __init__(
         self.cached_block_hash_to_block: Dict[BlockHashType, Dict[
             int, KVCacheBlock]] = defaultdict(dict)
 
+        # NOTE: this is used in `self._get_cached_block` for better performance
+        self.cached_block_hash_to_block_get = \
+            self.cached_block_hash_to_block.get
```

Comment:
Didn't know this before for sure so thanks for adding it!
Meanwhile, I feel this makes the code look a bit more complicate, do you think we can do something similar to the following to hide the complexity?

```python
class CachedGetDict(dict)
    def __init__(self, *args, **kwargs):
        self.cached_get = super().get

    def get(self, key, default):
        return self.cached_get(key, default)
```